csv_handler.cpp

```diff
@@ -14,7 +14,11 @@ bool is_numeric(const std::string& str) {
 
 int read_csv(const char *filename, bool is_gzip, float **mat_ptr, int *nr_ptr, int *nc_ptr) {
 	file_reader reader(is_gzip);
-	if (!reader.open(filename, is_gzip)) return FILE_OPEN_FAILED;
+	try{
```

Comment:
It would be better to keep the spacing policy consistent with the original code.

Reply:
Modified the callee side to properly return a Boolean value.